tools/vdd_sealed_channel_stress.ps1

```
@@ -0,0 +1,143 @@
+param(
+  [int]$Iterations = 5,
+  [int]$Monitor = 0,
+  [int]$Slots = 3,
+  [int]$DelaySeconds = 3,
+  [string]$ProbePath = "",
+  [string]$SunshineLog = "C:\Program Files\Sunshine\config\sunshine.log",
+  [string]$TriggerMonitorScript = "C:\Users\mohaha\Program\github\Virtual-Display-Driver\tools\vdd_capture_test\trigger_monitor.ps1",
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'TriggerMonitorScript' has been declared but not used.

(PSReviewUnusedParameter)